package.json

```diff
@@ -28,7 +28,7 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^16.0.0"
```

Comment:
what about 15.  I think we need to support that still.  

This might work better:

```
    "react": ">=0.15.0 < 17.0.0"
```

Reply:
We can/should probably drop 14.x at this point.